tests/integration/test_kubectl_delete.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from helpers import (  # type: ignore[attr-defined]
+    IPAddressWorkaround,
+    check_prometheus_is_ready,
+)
+
+log = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+app_name = METADATA["name"]
+
+
+@pytest.mark.abort_on_fail
+async def test_deploy_from_local_path(ops_test, prometheus_charm):
+    """Deploy the charm-under-test."""
+    resources = {"prometheus-image": METADATA["resources"]["prometheus-image"]["upstream-source"]}
+    log.info("deploy local charm")
```

Comment:
```suggestion
    log.debug("deploy local charm")
```

let's try to keep the test logs clean 👍🏽

Reply:
Done